requirements.txt

```diff
@@ -1,3 +1,2 @@
-cwltool==3.0.20201203173111
```

Comment:
does this automatically install the correct cwltool?
also can we keep using the `[all]` option or should it only be the `[cwl]` option?